src/index.ts

```diff
@@ -127,7 +127,15 @@ export function configure(rootConfig: UrlCatConfiguration) {
     pathTemplateOrParams: string | ParamMap,
     maybeParams: ParamMap = {}, config: UrlCatConfiguration = {}
   ): string =>
-    urlcat(baseUrlOrTemplate, pathTemplateOrParams, maybeParams , { ...rootConfig, ...config });
+    urlcat(baseUrlOrTemplate, pathTemplateOrParams, maybeParams, { ...rootConfig, ...config });
+}
+
+function isValidRenderedPath(renderedPath: string, baseUrl: string, pathAndQuery: string): string {
+  if (renderedPath.length) {
+    return join(baseUrl, '/', pathAndQuery);
+  } else {
+    return join(baseUrl, '?', pathAndQuery);
+  }
 }
```

Comment:
@balazsbotond what about this?Is this correct way??

Reply:
Yes, I like this. There is small things though.

The fact that the name starts with `is...` suggests that this function returns `boolean` but it returns `string`. I think a better name would be `joinFullUrl` or something like that (feel free to suggest better alternatives).

Reply:
@balazsbotond I think ```joinFullUrl``` is nice 😄